tests/Kiota.Builder.Tests/KiotaBuilderTests.cs

```diff
@@ -6204,6 +6204,23 @@ public async Task IndexerTypeIsAccurateAndBackwardCompatibleIndexersAreAdded()
             application/json:
               schema:
                 $ref: '#/components/schemas/microsoft.graph.post'
+  /authors/{author-id}/posts:
```

Comment:
Please add another section with `/actors/{actor-id}/foo/baz` to test the solution supports multiple depths

Reply:
A 3rd case has been added in this test method